vendor/brave-ios/tests/ads_wrapper_test.mm

```diff
@@ -47,13 +45,10 @@ - (NSString *)stateStoragePath
 
 - (void)setUp
 {
-  self.braveCoreMain = [[BraveCoreMain alloc] init];
-  [self.braveCoreMain scheduleLowPriorityStartupTasks];
```

Comment:
Should we not comment out this code rather than removing until a solution is found? Thanks